test/unit/downloader.test.ts

```diff
@@ -102,10 +103,15 @@ describe('Downloader class', () => {
   })
 
   test('downloadContent successfully downloaded an image', async () => {
-    const { data: LondonDetail } = await Axios.get('https://en.wikipedia.org/api/rest_v1/page/mobile-sections/London')
-    const [imgToGet] = Object.values(LondonDetail.lead.image.urls)
-
-    const LondonImage = await downloader.downloadContent(imgToGet as string)
+    const { data: LondonHtml } = await Axios.get('https://en.wikipedia.org/api/rest_v1/page/html/London')
+    const doc = domino.createDocument(LondonHtml)
+    const imgToGet = Array.from(doc.querySelectorAll('[data-mw-section-id="0"] img'))[0]
+    let imgToGetSrc = ''
+    if (imgToGet.getAttribute('src')) {
```

Comment:
I don't understand why there is a `if` here? Don't assume the HTML could change.

Reply:
It's a good practice to check whether DOM element has a specific attribute before calling it directly.